doc/news/_preparation_next_release.md

```diff
@@ -50,8 +50,13 @@ docker run -it elektra/elektra
 
 ### New Backend
 
+The entire logic for backends has been rewritten.
+
+<!-- TODO [new_backend]: finish release notes -->
```

Comment:
Please add more information here, this is also important for us to get a grasp what changed (not only for release).